src/index.html

```diff
@@ -11,7 +11,193 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="field-set">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname"> Surname: </label>
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+          >
+
+        </div>
+
+        <div class="form-field">
+          <label for="name">Name: </label>
+          <input
+            type="text"
+            id="name"
+            name="name"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="age">How old are You? </label>
+          <input
+            type="number"
+            id="age"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="birth">Full date of birth: </label>
+          <input
+            type="date"
+            id="birth"
+            name="birth"
+          >
+        </div>
+
+        <div class="form-label">
+          <label for="terms">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            id="terms"
+            name="terms"
+          >
+        </div>
+      </fieldset>
+
+
+      <fieldset class="field-set">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div>
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            id="password"
+            name="password"
+            minlength="5"
+            maxlength="32"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="field-set">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+        Do you love cats?
+
+          <label>
+            <input
+            type="radio"
+            name="radioGroup"
+            value="yes"
+            >
```

Comment:
Still, indentations are off
```suggestion
            <input
              type="radio"
              name="radioGroup"
              value="yes"
            >
```